contracts/oracle/ICollateralizationOracle.sol

```diff
@@ -9,11 +9,10 @@ interface ICollateralizationOracle is IOracle {
 
     // ----------- Getters -----------
 
-    function isOvercollateralized() external view returns (bool);
-
-    function userCirculatingFei() external view returns (uint256);
+    // returns the PCV value, User-circulating FEI, and Protocol Equity, as well
+    // as a validity status.
+    function pcvStats() external view returns (uint256, uint256, int256, bool);
```

Comment:
Probably good to name the return values in interface so they are immediately recognizeable